server/plugin/plugin.go

```diff
@@ -152,6 +171,12 @@ func (p *Plugin) OnActivate() error {
 
 	registerGitHubToUsernameMappingCallback(p.getGitHubToUsernameMapping)
 
+	go func() {
+		err := p.forceResetAllMM34646()
+		if err != nil {
+			p.API.LogDebug("failed to reset user tokens", "error", err.Error())
+		}
```

Comment:
Why is this needed? Is the lazy migration when the clients connects not enough?

Reply:
I am not sure if the "old" tokens will still be resettable once they don't support that format; so 1/5 we should proactively reset? Conversely, I am not trysting a 1-time global backfill script to reset all of the tokens, so I put in the lazy user-specific reset as well.